source/plugins/ruby/in_kube_podinventory.rb

```diff
@@ -361,14 +361,14 @@ def parse_and_emit_records(podInventory, serviceRecords, continuationToken, batc
           eventStream = nil
         end
 
-        if containerInventoryStream.count > 0
-          $log.info("in_kube_podinventory::parse_and_emit_records: number of windows container inventory records emitted #{containerInventoryStream.count} @ #{Time.now.utc.iso8601}")
-          router.emit_stream(@containerInventoryTag, containerInventoryStream) if containerInventoryStream
-          if (!@@istestvar.nil? && !@@istestvar.empty? && @@istestvar.casecmp("true") == 0)
-            $log.info("kubeWindowsContainerInventoryEmitStreamSuccess @ #{Time.now.utc.iso8601}")
-          end
-          containerInventoryStream = nil
-        end
+        # if containerInventoryStream.count > 0
+        #   $log.info("in_kube_podinventory::parse_and_emit_records: number of windows container inventory records emitted #{containerInventoryStream.count} @ #{Time.now.utc.iso8601}")
```

Comment:
similarly here as well. this shouldnt be commented as this code will be used in legacy auth.